packages/access-client/src/agent-use-cases.js

```diff
@@ -264,9 +263,10 @@ export async function authorizeAndWait(access, email, opts = {}) {
  * @param {`${string}@${string}`} email
  * @param {object} [opts]
  * @param {AbortSignal} [opts.signal]
+ * @param {number} [opts.interval]
```

Comment:
I don't think we should need to pass this here.
Interval is only needed in order to construct a non-generic `opts.expectAuthorization: AuthorizationWaiter`.

It doesn't seem like it needs to be passed here, because it doesn't need to be passed to `authorizeAndWait` for many (any?) concrete implementation of `expectAuthorization`.

(and then there is no need to change the types)